### PR TITLE
Feature/luxe

### PR DESCRIPTION
**Pull request by GillesPl**
LuxeID integration

### CODE REVIEW

File: src/scripts/core/ds/DSClientModel.ts [hunk 1]
```diff
@@ -134,7 +134,8 @@ export class DSPlatformInfo extends BrowserInfo {
     constructor(public activated: boolean,
                 public managed: boolean,
```
Comment (jonasvekeman) [inline src/scripts/core/ds/DSClientModel.ts]: Managed flag can be removed?

Comment (GillesPl) [reply src/scripts/core/ds/DSClientModel.ts]: Did not see that one, will remove

**Comment by jonasvekeman** [comment]
Looks good to me, just one remark. Now we force the LuxeID to be used with CAN, we don't let the integrator choose between PIN or CAN. Not sure if this is an issue, maybe to check with @michallis .

**Comment by GillesPl** [comment]
Will check and reopen the ticket if there needs to be an implementation to support both Pin and Can

**Pull request merged**